src/main/java/seedu/address/logic/commands/applicant/DeleteApplicantCommand.java

```diff
@@ -12,6 +14,7 @@
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.applicant.Applicant;
+import seedu.address.model.interview.Interview;
 
 /**
  * Deletes a applicant identified using it's displayed index from the address book.
```

Comment:
Would it be clearer to add `Will delete any interviews that the applicant has as well` for this javadoc comment?